From 8b2114a1ea295a4576dd63091197633a90b20bd3 Mon Sep 17 00:00:00 2001
From: Marc Bourgoin <themard85@gmail.com>
Date: Fri, 8 Jun 2018 20:36:26 +0200
Subject: [PATCH 169/249] x2: update maintainers

* andr68rus and cr3pt have moved onto other devices
* Add me (please)

Change-Id: I4afbfa35e5241dac158c8e5c7d42cb1f1ef8caf4
---
 _data/devices/x2.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_data/devices/x2.yml b/_data/devices/x2.yml
index ee1d38f..11ef69d 100644
--- a/_data/devices/x2.yml
+++ b/_data/devices/x2.yml
@@ -18,7 +18,7 @@ height: 156.8 mm (6.17 in)
 image: x2.png
 install_method: fastboot_nexus
 kernel: android_kernel_leeco_msm8996
-maintainers: [tortel, andr68rus, cr3pt]
+maintainers: [tortel, ThEMarD]
 models: [LEX820, LEX821, LEX822, LEX829]
 name: Le Max2
 network:
-- 
2.17.1

